src/auth.py

```diff
@@ -413,46 +303,32 @@ def decode_access_token(
         now = int(time.time())
         one_year_seconds = 365 * 24 * 60 * 60
         return {
-            "https://deeporigin.io/claims/id/userHandle": "user-deeporigin-com",
-            "https://deeporigin.io/claims/id/userid": "google-apps|user@deeporigin.com",
-            "https://deeporigin.io/claims/id/email": "user@deeporigin.com",
-            "https://deeporigin.io/claims/id/emailVerified": True,
-            "userHandle": "user-deeporigin-com",
-            "userid": "google-apps|user@deeporigin.com",
-            "emailVerified": True,
-            "iss": "https://formicbio.us.auth0.com/",
-            "sub": "google-apps|user@deeporigin.com",
-            "aud": "https://os.deeporigin.io/api",
-            "iat": now,
             "exp": now + one_year_seconds,
-            "scope": "offline_access",
+            "iat": now,
+            "jti": "onrtro:11f26c41-4d64-15dc-cc13-bfbbfedbd744",
+            "iss": "https://local.deeporigin.io/realms/deeporigin",
+            "aud": ["do-app", "auth-service"],
+            "sub": "6b06d8f8-1f55-472c-a86c-f19651ba4b20",
+            "typ": "Bearer",
+            "azp": "pa-token-365d",
+            "sid": "3516d772-185c-6422-6bd8-5f7f34cf6a71",
+            "scope": "organizations:owner long-live-token",
+            "email_verified": True,
+            "name": "Doe User",
+            "given_name": "User",
+            "family_name": "Doe",
+            "email": "user@deeporigin.com",
         }
 
     if token is None:
-        tokens = get_tokens(env=env)
-        token = tokens["access"]
-
-    # Get the JWT header to extract the Key ID
-    unverified_header = jwt.get_unverified_header(token)
-    kid = unverified_header["kid"]
+        token = get_token(env=env)
 
-    # Get the public key using the Key ID
-    public_keys = get_public_keys(env=env)
-    for key in public_keys:
-        if key["kid"] == kid:
-            public_key = RSAAlgorithm.from_jwk(key)
-            break
-        raise DeepOriginException(f"Key ID {kid} not found in JWKS.")
+    # Get the JWT header
+    header = jwt.get_unverified_header(token)
```

Comment:
despite what sonar says, there's no risk here -- we're decoding the token without verifying signature but this isn't used for any serious logic -- the only thing we do with the decoded token is check if it's expired or not, and extract the name. no business logic is taken from this -- the server still checks the token for every action